scope

```diff
@@ -29,6 +29,34 @@ WEAVESCOPE_DOCKER_ARGS=${WEAVESCOPE_DOCKER_ARGS:-}
 COMMAND=$1
 shift 1
 
+# - The image embeds the weave script & Docker 1.3.1 client
+# - Docker versions prior to 1.5.0 do not support --pid=host
+MIN_DOCKER_VERSION=1.5.0
+
+check_docker_version() {
+    if ! DOCKER_VERSION=$(docker -v | sed -n -e 's|^Docker version \([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\).*|\1|p') || [ -z "$DOCKER_VERSION" ] ; then
```

Comment:
This is just copied from the weave script; I'll see if we can apply the same simplification there.